.github/workflows/on-push-do-docs.yml

```diff
@@ -0,0 +1,22 @@
+name: on-push-do-docs
+on:
+  push:
+jobs:
+  docs:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run MarkdownSnippets
+      run: |
+        dotnet tool install --global MarkdownSnippets.Tool
+        mdsnippets ${GITHUB_WORKSPACE}
+      shell: bash
```

Comment:
Seems a bit weird to me to use bash as the shell on a Windows runner.

Reply:
just what i picked. they are much the same from my point of view

Reply:
Maybe just use Linux as the OS for the job, rather than Windows?